### ISSUE TITLE
Act-Types in Act.csv

### ISSUE THREAD

**Issue by mira-gruenwald** [LenaHenningsen/ReadingData#168]
After working on #166  I have some questions:
@LenaHenningsen @BIGBUNNIEEEE  @duncdrum @inamoto4

Which Act-type do we add to Acts which just record that a book/text etc. was mentioned? Do we assume that the text was read in any case? 

What about Acts that refer to a person as act_object (e.g. A00021) ? Do we add AT04 ( to take as guide/ ideal) ?

**Comment by LenaHenningsen** [comment]
As to the „mentioning“ – I would be cautious with the assumption that the book was read. So we better leave it as vague as „mentioned“

 

As to the second question, I am not entirely sure what you mean…

 

Von: Mira Grünwald <notifications@github.com>
Antworten an: LenaHenningsen/ReadingData <reply@reply.github.com>
Datum: Montag, 22. Januar 2018 um 17:37
An: LenaHenningsen/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Mention <mention@noreply.github.com>
Betreff: [LenaHenningsen/ReadingData] Act-Types in Act.csv (#168)

 

After working on #166 I have some questions:
@LenaHenningsen @BIGBUNNIEEEE @duncdrum @inamoto4

Which Act-type do we add to Acts which just record that a book/text etc. was mentioned? Do we assume that the text was read in any case?

What about Acts that refer to a person as act_object (e.g. A00021) ? Do we add AT04 ( to take as guide/ ideal) ?

**Comment by inamoto4** [comment]
In the old Excel file we have included every text, which was mentioned in our sources, so we did not always a real "act" in the commentary cell.  The act_file so far insinuated something more active. If we say that "to mention" is an act, it might logically speaking a bit strechted, as so far we have only recorded acts which happened during the long 1970s, whereas "to mention" would usually refer to a book after 1980. It might again be hard to distinguish this from somebody writing in an autobiography that a certain book was mentioned during the time before 1980. 

"To mention" is definitely the (s)lightest act possible, but it would help us to include some titles. So I do not have any better idea. I would for sure go against "to read", as this is sth different.  

In the case of A00021 I do not see how that act described relates to Lu Xun. I would propose that we add sth like the "Complete works of Lu Xun", as the readers did not read Lu Xun but his books. So we should change that. Only if sbd says that he took the life of Lu Xun as his guide might we add a person as the act_object, but generally speaking this should not be possible in my opinion.
